keps/sig-apps/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -18,9 +18,9 @@ approvers:
   - "@msau42"
   - "@janetkuo"
 
-stage: beta
```

Comment:
I can't comment in the README.md file, which also needs updates:
1. update the `## Release Signoff Checklist` it doesn't match the new template https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md and make sure to check appropriate boxes in it. 
2. Would be good to update unit tests coverage, update links and list of integration tests and e2e tests. 
3. Can you point to upgrade/downgrade e2e test described in e2e section, subsection `##### Upgrade/downgrade & feature enabled/disable tests`
4. Do we have any data about customer feedback which is a requirement for `#### GA release`?
5. In `### Version Skew Strategy` the statement about apiserver and controller-manager not being a problem with version skew is not entirely correct. You can rollout apiserver and controller-manager separately, and that's how it's usually done, apiserver first and controller-manager follows. So this question needs an update. 
6. In `##### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` it's required to describe the full upgrade->downgrade test, since we rely on manual testing for now it has to be quite detailed. 
7. In `###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?` the answer should be no, since you're not removing/deprecating any fields. 
8. In `##### Will enabling / using this feature result in increasing size or count of the existing API objects?` would be nice to make sure you estimate the increase in size in bytes. 
9. `## Implementation History` update is required.

Reply:
I still owe you finishing (1) (conformance tests) & (6).

(1) almost done (modulo the conversation on https://github.com/kubernetes/kubernetes/pull/127967)

(2) done

(3) the comment still holds --- I don't know how to do upgrade/downgrade tests. Perhaps there's now a mechanism to do this, in which case any pointer would be welcome.

(4) Added details to GA release section in Graduation Criteria

(5) done

(6) in progress

(7) done

(8) done

(9) done

Reply:
@mattcary Would you ping us again when you finish (1) and (6)? We'll then take another look

Reply:
Will do. @soltysh and I discussed (1) offline, and have decided we're good on the conformance tests. So only (6) is remaining.

Reply:
Yup, Matt and I are in contact on slack, I'll be watching here as well to make sure that once the upgrade/rollback tests is added we're good to go.